EnglishFilter/sections/specific.txt

```diff
@@ -307,6 +307,8 @@ freehdinterracialporn.in###video > div.border + div[class]
 freehdinterracialporn.in##.my-ad
 ||moanmyip.com/images/removemyphone.png
 justalternativeto.com##.justa-adlabel
+pornhat.*##.top_spot
+pornhat.*##.bn-new
```

Comment:
Does this change something on your side? I see no visual difference.

Reply:
I don't see the need for `pornhat.*##.bn-new`, but `pornhat.*##.top_spot` is required on my end.